src/style.scss

```
@@ -125,5 +125,133 @@ $font-normal: 500;
     }
   }
 
-  // 도전 금액 input 페이지
+  // 도전 금액 입력 페이지
+  #app-challenge-input {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: $color-white;
+    height: 40rem;
+    border-radius: 1.5rem;
+    padding: 1.5rem 1rem;
+    font-weight: $font-bold;
+  }
+
+  .header-challenge-input {
+    height: 10%;
+
+    .back-to-main-button {
+      font-size: $font-large;
+    }
+  }
+
+  .main-challenge-input {
+    height: 80%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+
+    p {
+      font-size: $font-large;
+      color: $color-green;
+      margin: 1rem 0;
+    }
+
+    input {
+      font-size: $font-large;
+      color: $color-green;
+      border: 0;
+      border-bottom: 0.2rem solid $color-green;
+      background-color: $color-white;
+      margin-left: 1rem;
+
+      &::placeholder {
+        color: $color-green;
+        opacity: 30%;
+        font-weight: $font-bold;
+      }
+    }
+  }
+
+  .footer-challenge-input {
+    height: 10%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .input-button {
+      width: 100%;
+      height: 100%;
+      line-height: 60px;
+      background-color: $color-green;
+      border-radius: 1.5rem;
+      text-align: center;
+      font-size: $font-medium;
+    }
+  }
+
+  // 소비 품목 입력 페이지
+  #app-item-input {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: $color-white;
+    height: 40rem;
+    border-radius: 1.5rem;
+    padding: 1.5rem 1rem;
+    font-weight: $font-bold;
+  }
+
+  .header-item-input {
+    height: 10%;
+
+    .back-to-main-button {
+      font-size: $font-large;
+    }
+  }
+
+  .main-item-input {
+    height: 80%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+
+    p {
```

```suggestion
    > p {
```

다른 p가 겹치면 style오류가 생길 수 있어서 요렇게!

---

아하, 넵 확인했습니다 :)!